animation_nodes/nodes/gpencil/gp_object_output.py

```diff
@@ -0,0 +1,114 @@
+import bpy
+from bpy.props import *
+from ... base_types import AnimationNode, VectorizedSocket
+
+class GPObjectOutputNode(bpy.types.Node, AnimationNode):
+    bl_idname = "an_GPObjectOutputNode"
+    bl_label = "GP Object Output"
+    errorHandlingType = "EXCEPTION"
+
+    useLayerList: VectorizedSocket.newProperty()
+
+    def create(self):
+        socket = self.newInput("Object", "Object", "object")
+        socket.defaultDrawType = "PROPERTY_ONLY"
+        socket.objectCreationType = "GPENCIL"
+        self.newInput(VectorizedSocket("GPLayer", "useLayerList",
+            ("Layer", "layer"), ("Layers", "layers")))
+        self.newOutput("Object", "Object", "object")
+
+    def getExecutionFunctionName(self):
+        if self.useLayerList:
+            return "execute_LayerList"
+        else:
+            return "execute_Layer"
+
+    def execute_Layer(self, object, layer):
```

Comment:
You seem to have a lot of code duplication between `execute_Layer` and `execute_LayerList`. Can you refactor that?

Reply:
It is changed.